src/routes/NoteEditRoute.test.js

```diff
@@ -0,0 +1,43 @@
+import '__mock__/';
+import { render, screen } from '@testing-library/react';
+import { historyData } from '../../test/jest/fixtures/historyData';
+import NoteEditRoute from './NoteEditRoute';
+
+jest.mock('react-router', () => ({
+  ...jest.requireActual('react-router'),
+  Redirect: () => <div>Request</div>
+}));
+
+const locationData = historyData.location;
+const match = {
+  params: {
+    noteId: 'editNoteRouteID'
+  },
+  path: 'editPath',
+  url: '{{ env.FOLIO_MD_REGISTRY }}/_/proxy/modules'
+};
+
+const renderNoteEditRoute = (locationProps, historyProps) => render(
+  <NoteEditRoute location={locationProps} history={historyProps} match={match} />
+);
+describe('NoteEditRoute', () => {
+  it('NoteEditPage should render when location.state is not empty', () => {
+    renderNoteEditRoute(locationData, historyData);
+    expect(screen.getByText('NoteEditPage')).toBeInTheDocument();
+  });
+  it('Request page should render when location.state is empty', () => {
+    const historyProp = {
+      ...historyData,
+      location : {
+        hash: '',
+        key: '9pb09t',
+        pathname: '/users/notes/new',
+        search: '',
+        state: ''
+      },
```

Comment:
Looks like some properties are not necessary here

```suggestion
      location : {
        state: ''
      },
```

Reply:
Hi @artem-blazhko , the following properties are added in order to avoid console warnings. Below is the attached screenshot for reference:
![image](https://github.com/folio-org/ui-requests/assets/100114810/69788b5b-37c2-4329-b7b3-4c1300bb7351)